packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -1,42 +1,51 @@
 /* eslint-disable class-methods-use-this */
 const expect = require('expect');
-const { accessibility } = require('../commands/validates');
+const { accessibility, element } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
-const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
-
-const hideInputCaret = require('../commands/hide-input-caret');
-const { element } = require('../commands/validates');
+const {
+  describeViewports,
+  getViewports,
+  hideInputCaret,
+  setViewport,
+} = require('../commands/utils');
 
 class TerraService {
   constructor(options = {}) {
-    this.serviceOptions = {
-      theme: 'terra-default-theme',
-      ...options,
-    };
+    this.serviceOptions = options;
   }
 
+  /**
+   * Service hook executed before worker processes are launched.
+   * @param {object} config - The WebdriverIO configuration object.
+   */
   onPrepare(config) {
-    // Add the configurations at the top level service options to the Terra Service options.
+    const { serviceOptions } = config;
+
     this.serviceOptions = {
+      theme: 'terra-default-theme',
       selector: '[data-terra-test-content] *:first-child',
       ...this.serviceOptions,
-      ...config.serviceOptions,
+      ...serviceOptions,
     };
   }
 
+  /**
+   * Service hook executed prior to initializing the webdriver session.
+   */
   beforeSession() {
-    global.Terra = {
-      /*
-       This command must be defined in the beforeSession hook instead of together with the other Terra custom commands in the
-       before hook. The reason being webdriverio v6 now executed the describe block prior to running the before hook.
-       Therefore, this command needs to be defined before the test starts in the testing life cycle.
-
-       Reference: https://github.com/webdriverio/webdriverio/issues/6119
-      */
-      describeViewports,
-
-      serviceOptions: this.serviceOptions,
-    };
+    global.Terra = {};
+
+    /**
+     * This command must be defined in the beforeSession hook instead of together with the other Terra custom commands in the
+     * before hook. The reason being WebdriverIO v6 now executes the describe block prior to running the before hook.
+     * Therefore, this command needs to be defined before the test starts in the testing life cycle.
+     *
+     * Reference: https://github.com/webdriverio/webdriverio/issues/6119
+     */
+    global.Terra.describeViewports = describeViewports;
+
+    // Add the service options to the global.
+    global.Terra.serviceOptions = this.serviceOptions;
```

Comment:
Why do these need to be added to the Terra global?

Reply:
This exposes the service options globally so the utility functions can reference them. This is existing functionality:

- [Related discussion/PR that introduced this change](https://github.com/cerner/terra-toolkit/pull/464#discussion_r516734946)
- [Example of where it's used](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/commands/viewport-helpers.js#L50)

Long term we can probably bind service options to the function to pass them along, but it can create a long chain of information for certain functions that call other functions. I didn't make any changes to exposing the options globally on this PR